source/base/mpi.cc

```diff
@@ -542,8 +509,20 @@ namespace Utilities
     }
 
 
-  } // end of namespace MPI
 
+    bool job_supports_mpi ()
+    {
+#ifdef DEAL_II_WITH_MPI
+      int MPI_has_been_started = 0;
+      MPI_Initialized(&MPI_has_been_started);
+
+      return (MPI_has_been_started > 0);
+#else
+      return false;
+#endif
+    }
+#include "mpi.inst"
```

Comment:
Can you add a couple of empty lines before this statement? It's hard to see that it's not part of the preceding function.